std/string.d

```diff
@@ -2224,6 +2224,13 @@ S capitalize(S)(S s) @trusted pure
     return changed ? cast(S)retval : s;
 }
 
+///
+unittest
```

Comment:
I'll add pure, even though it does not show up in the output. trusted has no meaning